docs/specs/ops.yml

```diff
@@ -282,3 +282,82 @@ outputs:
   .errors.log: |
     {"message_severity":"error","code":"violate-schema","message":"Error converting value \"a/b/.openpublishing.redirection.test.json\" to type 'System.Collections.Generic.HashSet`1[Microsoft.Docs.Build.PathString]'.","file":".openpublishing.publish.config.json","line":3,"column":66}
     {"message_severity":"error","code":"violate-schema","message":"Error converting value \"a/b/.openpublishing.redirection.test.json\" to type 'System.Collections.Generic.HashSet`1[Microsoft.Docs.Build.PathString]'.","file":".openpublishing.publish.config.json","line":3,"column":66}
+
+---
+# provide source url in .publish.json for content, toc and resource pages but not redirection pages.
+repos:
+  https://github.com/test/repo:
+  - files:
+      .openpublishing.publish.config.json: |
+        {
+          "docsets_to_publish": [{"build_source_folder":"a"}],
+        }
+      .openpublishing.redirection.json: |
```

Comment:
See tests in `redirection.yml` to use `redirections.yml` to setup redirection. Use `openpublishing.xyz` file only for docs/ops specific features.

Reply:
@yufeih got it, I just update code, please help review again.